client-sdk/ts-web/rt/playground/src/consensus.js

```diff
@@ -213,21 +213,23 @@ export const playground = (async function () {
         let depositEvent = null;
         const depositEventVisitor = new oasisRT.event.Visitor([
             oasisRT.consensusAccounts.moduleEventHandler({
-                [oasisRT.consensusAccounts.EVENT_DEPOSIT_CODE]: (e, depositEv) => {
-                    console.log('polled deposit event', depositEv);
-                    const eventFromBech32 = oasis.staking.addressToBech32(depositEv.from);
-                    if (eventFromBech32 !== addrAliceBech32) {
-                        console.log('address mismatch');
-                        return;
+                [oasisRT.consensusAccounts.EVENT_DEPOSIT_CODE]: (e, depositEvs) => {
```

Comment:
Could this be changed such that the changes wouldn't be needed (e.g. that the visitor/handler would handle this and just call the function multiple times instead of requiring the developer to always iterate over events)?

Reply:
Done, was actually easier than the previous change